static/skywire-manager-src/src/app/components/vpn/pages/vpn-error/vpn-error.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Subscription } from 'rxjs/internal/Subscription';
+
+import { VpnAuthGuardService } from 'src/app/services/vpn-auth-guard.service';
+import { VpnClientService } from 'src/app/services/vpn-client.service';
+
+/**
+ * Errors VpnErrorComponent can show.
+ */
+enum KnownProblems {
+  UnableToConnectWithTheVpnClientApp = 'unavailable',
+  NoLocalVisorPkProvided = 'pk',
+  InvalidStorageState = 'storage',
+  LocalVisorPkChangedDuringUsage = 'pkChange',
+}
+
+/**
+ * Page for showing an important error which made the VPN client stop working.
+ * For indicating the problem, use the "problem" var in the querystring. You can check the valid
+ * values in the KnownProblems enum. If no value is provided,
+ * KnownProblems.UnableToConnectWithTheVpnClientApp is used.
+ */
+@Component({
+  selector: 'app-vpn-error',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skywire/pullRequest?prid=7163595)
```suggestion
  selector: "app-vpn-error",
```